src/github.com/robertkrimen/otto/.gitignore

```diff
@@ -1,5 +0,0 @@
-/.test
```

Comment:
Right. I was just using it previously to test the validity of the updated PAC file.